### ISSUE TITLE
Support for Python Virtual Environments

### ISSUE THREAD

**Issue by witoldpiet** [opentap/OpenTap.Python#159]
Is it possible to extend the Python plugin with support for Python Virtual Environments?

**Comment by rmadsen-ks** [comment]
Windows: How to add a virtual environment:
```py -m venv [folder.venv]```

The folder contains the following files by default:
```
04/30/2024  02:26 PM    <DIR>          Include
04/30/2024  02:26 PM    <DIR>          Lib
04/30/2024  02:26 PM               325 pyvenv.cfg
04/30/2024  02:27 PM    <DIR>          Scripts
               1 File(s)            325 bytes
````

pyvenv.cfg contains information about the environment. We should be able to detect the actual python installation based on that.

**Comment by rmadsen-ks** [comment]
This has been fixed in 3.2 beta
